src/locales/en.json

```diff
@@ -65,7 +65,6 @@
     "solde": "Balance",
     "account_number": "Account number",
     "account_name": "Account name",
-    "add_bank": "Add a bank",
```

Comment:
It was already at the right place in french translations?

Reply:
yes I clean it [here](https://github.com/cozy/cozy-banks/commit/891b25b54897dfd4fd20911cfda9ec7ea80403e8). I will fixup this commit ;)